### DISCUSSION TITLE
[solved] error cloning into /tmp/reaction-[random_string]

### DISCUSSION THREAD

**Discussion by parallax-kal** [reactioncommerce/reaction#6678] [Q&A]
![image](https://user-images.githubusercontent.com/99996697/205018784-ca9d7312-be88-4384-89c0-88661602845c.png)
I encountered this error when I was running reaction create-project api [name_project]

but I realized that I have to  follow this video https://youtu.be/Qu215s3hYnQ

**Comment by brent-hoover** [comment]
@the-coder-kalisa thanks a lot for this!
